src/components/todo/TodoList.tsx

```diff
@@ -0,0 +1,16 @@
+import { TodosContext } from 'contexts/todosContext';
+import { useContext } from 'react';
+
+import TodoItem from './TodoItem';
```

Comment:
이 친구만 상대적 경로인게 아쉽습니다
일관성을 위해 절대경로로 바꿔주세요
```suggestion
import TodoItem from 'components/todo/TodoItem';
```